mobile/lib/main.dart

```diff
@@ -156,6 +157,8 @@ class ImmichAppState extends ConsumerState<ImmichApp>
 
         ref.read(iOSBackgroundSettingsProvider.notifier).refresh();
 
+        ref.read(todayProvider.notifier).update();
```

Comment:
I think you only need to perform `ref.invalidate(memoryFutureProvider);` in this page, and it would solve the issue

Reply:
Ah, that's *MUCH* cleaner! I think you are right.